src/lib/connectors/io-redis.js

```diff
@@ -4,7 +4,7 @@ const IORedis = require('ioredis');
 const config = require('../../../config');
 
 // Note: this limit will need increasing if further Bull MQ job queues are added
-const maxListenerCount = 20;
+const maxListenerCount = 23;
```

Comment:
Wondering if this should be stored in and be fetched from `config.redis.maxListeners` because I have a feeling we will end up changing it every now and then.

Reply:
Done